lib/smoke-test.js

```diff
@@ -1,277 +0,0 @@
-'use strict';
```

Comment:
OMG, I'm getting all emotional seeing this file die :cry:

Reply:
Death is just another path, one that we all must take. The grey rain-curtain of this world rolls back, and all turns to silver glass, and then you see it.

Reply:
(sorry I was watching Lord of the Rings the other day)

Reply:
Would you like to borrow my scythe and cowl?